webDemo/src/App.tsx

```diff
@@ -399,30 +399,10 @@ function App() {
 }
 
 const styles = StyleSheet.create({
-  header: {
-    padding: 20
-  },
   componentContainer: {
-    backgroundColor: '#F5FCFF',
-    width: '100%',
-    alignItems: 'center',
-    justifyContent: 'center',
-    borderColor: 'white',
-    borderBottomWidth: 5,
-    paddingBottom: 20,
-    paddingTop: 20
-  },
-  compTitle: {
-    fontWeight: 'bold',
-    fontSize: '1rem',
-    marginVertical: 4,
-    textAlign: 'center'
-  },
-  title: {
-    fontWeight: 'bold',
-    fontSize: '1.5rem',
-    marginVertical: '1em',
-    textAlign: 'center'
```

Comment:
Seems like the typography modifiers don't work well on the web (mainly the font-weight I guess).
I suggest to returned it back to use the stylesheet, unless you want to change the design :)

Reply:
Returned to the stylesheet, in the redesign I'll use our modifiers.